site/_posts/2022-09-09-what-is-webauthn-why-do-you-care.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: blog-post
+title: What is WebAuthn and why do you care?
+description: WebAuthn is a new standard which secures user accounts and improves user experience. What could be better?
+author: Dan Moore
+image: blogs/sso-explanation/how-sso-works.png 
+category: blog
+tags: topic-single-sign-on
+excerpt_separator: "<!--more-->"
+---
+
+WebAuthn is a new way for people to authenticate themselves to web applications. It's a widely supported standard years in the making. All major browsers work with it, which makes it easy for developers to incorporate WebAuthn into websites.
+
+WebAuthn allows users to leverage the power of biometric, device and other authentication methods via a simple, consistent, browser-native user experience.
```

Comment:
Consistency between browsers is relative. I think that using WebAuthn in Safari feels different than Chrome, but it's close enough that users shouldn't feel lost.

What does "device authentication method" mean? Not sure whether it's worth spending a ton of time here if it's explained in more detail before. I'll circle back.